src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.engine.backend
+
+import cromwell.binding.values.WdlValue
+import cromwell.binding.{Call, WdlStandardLibraryFunctions, WorkflowDescriptor}
+
+import scala.util.Try
+
+trait BackendCall {
```

Comment:
I am going to update/add code comments at some point this weekend.  I'll have to think about the refactoring you described, I don't think I fully understand what you're going for, we can talk on monday about it.